pkg/apiserver/storage/file/file.go

```diff
@@ -15,9 +15,9 @@ import (
 	"sync"
 	"time"
 
-	"github.com/bwmarrin/snowflake"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/meta"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
```suggestion
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
we use this in most other places

Reply:
updated in https://github.com/grafana/grafana/pull/90151/commits/7d173efe465bffe194460deea2ae65a75c142126